.semaphore/semaphore.yml

```diff
@@ -28,6 +27,7 @@ blocks:
           value: "-DskipTests -Djacoco.skip=true"
       prologue:
         commands:
+          - echo $SEMAPHORE_WORKFLOW_ID
```

Comment:
isn't that ID already part of the dashboard when we open the CI logs

Reply:
you can get the workflow id by clicking the CLI button in the semaphore UI
<img width="348" alt="Screen Shot 2022-10-10 at 5 27 13 PM" src="https://user-images.githubusercontent.com/815147/194971985-69ec7fb9-5aeb-4e9e-9046-4778ac543ca1.png">

Reply:
This is an inherited change